mods/player_api/api.lua

```diff
@@ -60,37 +58,46 @@ function player_api.set_model(player, model_name)
 			eye_height = 1.625,
 		})
 	end
-	player_model[name] = model_name
+	player_data.model = model_name
```

Comment:
`player_data` is local to the function, surely this needs to write data to `players[player_name]`?

Reply:
Not at all. `players[player_name]` is a table. `player_data.model` is the same as `players[player_name].model`.

Reply:
Yes, i just remembered, tables in Lua contain references to values, so i assume altering a copy of a table alters the original table.